prime-angular-frontend/src/app/modules/enrolment/pages/care-setting/care-setting.component.ts

```diff
@@ -190,12 +193,16 @@ export class CareSettingComponent extends BaseEnrolmentProfilePage implements On
   }
 
   protected nextRouteAfterSubmit() {
-    const jobs = this.enrolmentFormStateService.jobsForm.get('jobs').value as Job[];
+    const oboSites = this.enrolmentFormStateService.jobsForm.get('oboSites').value as OboSite[];
+    const certifications = this.enrolmentFormStateService.regulatoryFormState.certifications;
 
     let nextRoutePath: string;
     if (!this.isProfileComplete) {
       nextRoutePath = EnrolmentRoutes.REGULATORY;
-    } else if (jobs?.length) {
+    } else if (oboSites?.length) {
+      nextRoutePath = EnrolmentRoutes.JOB;
+    } else if (certifications.length === 0 && oboSites?.length === 0) {
```

Comment:
You're using JS language syntax for falsey above with a null/length check on `oboSites?.length`, but then change it to an explicit check on 0.  I know you don't like JS language syntax, but you're using it then not using it, but it would be more consistent if you had: `!certifications?.length && !oboSites?.length`